controllers/mover/syncthing/mover.go

```diff
@@ -509,6 +525,26 @@ func (m *Mover) ensureDeployment(ctx context.Context, dataPVC *corev1.Persistent
 				},
 			},
 		}
+
+		if m.privileged {
+			podSpec.Containers[0].Env = append(podSpec.Containers[0].Env, corev1.EnvVar{
+				Name:  "PRIVILEGED_MOVER",
+				Value: "1",
+			})
+			podSpec.Containers[0].SecurityContext.Capabilities.Add = []corev1.Capability{
+				"DAC_OVERRIDE", // Read/write all files
+				"CHOWN",        // chown files
+				"FOWNER",       // Set permission bits & times
+			}
+			podSpec.Containers[0].SecurityContext.RunAsUser = pointer.Int64(0)
+			podSpec.Containers[0].SecurityContext.RunAsNonRoot = pointer.Bool(false)
```

Comment:
this is the really strange thing - It doesn't seem to be needed for rclone/restic.  It took me a while to figure out that the `RunAsNonRoot` not being set caused an issue with syncthing only.  My theory is that it's a difference between Job and Deploy since syncthing uses a deployment.  Do you think we should do it in all 3 to be consistent?